src/Evernote/Auth/OauthHandler.php

```diff
@@ -116,7 +114,8 @@ protected function getTemporaryCredentials()
             CURLOPT_HTTPHEADER     => $this->formatHeaders($headers),
             CURLOPT_POSTFIELDS     => http_build_query($arguments, '', '&'),
             CURLOPT_HEADER         => true,
-            CURLOPT_RETURNTRANSFER => true
+            CURLOPT_RETURNTRANSFER => true,
+			CURLOPT_SSL_VERIFYPEER => false
```

Comment:
Sorry, change is irrelevant to the fix. Accidentally pushed. Will look into removing the commit now.